tests/test_edgeql_introspection.py

```diff
@@ -939,6 +939,123 @@ class TestIntrospection(tb.QueryTestCase):
         self.assertIsNotNone(result[0].links[0].target.id)
         self.assertIsNotNone(result[0].properties[0].target.id)
 
+    async def test_edgeql_introspection_meta_default_01(self):
+        await self.assert_query_result(
+            r'''
+                WITH MODULE schema
+                SELECT ObjectType {
+                    name,
+                    is_abstract
+                }
+                FILTER .name LIKE 'test::%'
+                ORDER BY .name;
+            ''',
```

Comment:
I don't like tests like this because they break when we add another type to a schema. Select explicitly the names you're testing.

Reply:
I've trimmed the test to a couple specific types.